docs/SOPs/2-updating_lcia.md

```diff
@@ -0,0 +1,25 @@
+# Updating Life Cycle Impact Assessment Methods using LCIA Formatter
+
+## Purpose
+
+This SOP documents procedures for making updates or adding new LCIA methods to the Federal LCA Commons (FLCAC).
```

Comment:
Consider making it clear that this SOP is to be carried out by the Data Curator as it is going to be posted somewhere that the public can access it.